test/test_tests.py

```diff
@@ -1,44 +1,44 @@
-import os
-from unittest.mock import call, MagicMock, patch
+from unittest.mock import MagicMock, patch
 
-from exasol_integration_test_docker_environment.cli.commands.build_test_container import build_test_container
-from exasol_script_languages_container_tool.cli.commands import build, run_db_test, push, security_scan
+import pytest
 
 from exasol_script_languages_container_ci.lib.ci import ci
+from test import exaslct_calls
 
-from test.fixtures import tmp_test_dir, click_stub, test_env, patch_printfile
+from test.fixtures import tmp_test_dir, click_stub, patch_printfile
+from test.test_env import test_env
 
+testdata = [
```

Comment:
I think, we should explain how the test data is structured

Reply:
Done